draft-ietf-httpbis-cache-latest.xml

```diff
@@ -1080,13 +1080,38 @@
    response, with the following exceptions:
 </t>
 <ul>
-   <li>The exceptions to header field storage in <xref
-   target="storing.fields"/> also apply to header field updates.</li>
-   <li>Caches &MUST-NOT; update the following header fields:
-   <x:ref>Content-Encoding</x:ref>, <x:ref>Content-Length</x:ref>,
-   Content-MD5 (<xref target="RFC2616" x:fmt="of" x:sec="14.15"/>),
-   <x:ref>Content-Range</x:ref>, <x:ref>ETag</x:ref>.</li>
+   <li>Header fields excepted from storage in <xref target="storing.fields"/>,</li>
+   <li>Header fields that the cache's stored representation of the response depends upon, as described below,</li>
+   <li>Header fields that are automatically processed and removed by the recipient, as described below, and</li>
+   <li>The Content-Length header field.</li>
 </ul>
```

Comment:
I'm a bit nervous that we seem to loose a concrete list of field names in a normative requirement.

Reply:
Discussed on call.